apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -453,6 +466,118 @@ export class LemonadePipeline implements BasePipeline {
+            ticket: {
+              argumentType: ArgumentTypeName.Object,
+              value: {
+                // The fields below are not signed and are used for display purposes.
+                eventName: badgeConfig.eventName,
+                ticketName: badgeConfig.productName ?? "",
+                checkerEmail: undefined,
+                imageUrl: badgeConfig.imageUrl,
+                imageAltText: undefined,
+                // The fields below are signed using the passport-server's private EdDSA key
+                // and can be used by 3rd parties to represent their own tickets.
+                eventId, // The event ID uniquely identifies an event.
+                productId, // The product ID uniquely identifies the type of ticket (e.g. General Admission, Volunteer etc.).
+                ticketId, // The ticket ID is a unique identifier of the ticket.
+                timestampConsumed: 0,
+                timestampSigned: Date.now(),
+                attendeeSemaphoreId: "",
+                isConsumed: false,
+                isRevoked: false,
+                ticketCategory: TicketCategory.Generic,
+                attendeeName: "",
+                attendeeEmail: ""
+              }
+            }
+          })
+        );
+      })
+    );
+
+    return badgePCDs.filter((pcd) => !!pcd) as SerializedPCD<EdDSATicketPCD>[];
+  }
+
+  private async getReceivedContactsForEmail(
+    email: string
+  ): Promise<SerializedPCD<EdDSATicketPCD>[]> {
+    const contacts = await this.contactDB.getContacts(this.id, email);
+    return Promise.all(
+      contacts.map(async (contact) => {
+        return await EdDSATicketPCDPackage.serialize(
+          await EdDSATicketPCDPackage.prove({
+            id: {
+              argumentType: ArgumentTypeName.String,
+              value: `${contact}->${email}`
+            },
+            privateKey: {
+              argumentType: ArgumentTypeName.String,
+              value: this.eddsaPrivateKey
+            },
+            ticket: {
+              argumentType: ArgumentTypeName.Object,
+              value: {
+                // The fields below are not signed and are used for display purposes.
+                eventName: "Contact Card",
+                ticketName: "",
+                checkerEmail: undefined,
+                imageUrl:
+                  "https://t4.ftcdn.net/jpg/00/97/58/97/360_F_97589769_t45CqXyzjz0KXwoBZT9PRaWGHRk5hQqQ.jpg",
+                imageAltText: undefined,
+                // The fields below are signed using the passport-server's private EdDSA key
+                // and can be used by 3rd parties to represent their own tickets.
+                ticketId: randomUUID(), // The ticket ID is a unique identifier of the ticket.
+                eventId: randomUUID(), // The event ID uniquely identifies an event.
+                productId: randomUUID(), // The product ID uniquely identifies the type of ticket (e.g. General Admission, Volunteer etc.).
```

Comment:
can this be uuidv5() of the `eventName` and `attendeeEmail`?

Reply:
sure